src/test/java/org/opentripplanner/generate/doc/framework/DocBuilder.java

```diff
@@ -142,12 +111,12 @@ public void addExample(String comment, JsonNode body) {
       """.formatted(comment, json));
   }
 
-  private static String prettyPrintJson(JsonNode body) {
-    try {
-      return PRETTY_PRINTER.writeValueAsString(body);
-    } catch (JsonProcessingException e) {
-      throw new RuntimeException(e);
-    }
+  public void addUpdaterExample(String comment, JsonNode node) {
+    var updaters = MAPPER.createArrayNode();
+    updaters.add(node);
+    var root = MAPPER.createObjectNode();
```

Comment:
The MAPPER is not a constant it can be changed, so a way to encapsulate it is the make MAPPER private and create a ExampleBuilder in the `TemplateUtil` that you can use here. Something like this:
```
  val exampleNode = TemplateUtil.jsonExampleBuilder(node).wrapInArray().wrapInObject("updaters").build();
  addExample(comment, exampleNode);
```